### PR TITLE
Terraform: override App Service Cipher Suite

### LINKED ISSUE #771: Limit to secure cipher suites
Jemurai found that [we are using insecure cipher suites](https://docs.google.com/spreadsheets/d/12BIe2hmxGIFsq8_rdpbeiemdyVz0qYld/edit#gid=1303467638).

## To Reproduce

1. [Scan with SSLLabs](https://www.ssllabs.com/ssltest/analyze.html?d=test-benefits.calitp.org)
2. Click on one of the IPs
3. Under `Cipher Suites`, see `TLS_DHE_RSA_WITH_AES_*`

## Expected behavior

Following steps above, `TLS_DHE_RSA_WITH_AES_*` _shouldn't_ appear.

## Screenshots

n/a

## Additional context

- https://docs.microsoft.com/en-us/azure/app-service/environment/app-service-app-service-environment-custom-settings#change-tls-cipher-suite-order
- https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/app_service_environment_v3

Possible there's an existing environment that needs to be [imported](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/app_service_environment_v3#import).

### PR DESCRIPTION
**Pull request by thekaveman**
Closes #771 

We're currently getting an error trying to `terraform apply` these changes:

> Error: checking for presence of existing App Service Environment: (Hosting Environment Name "ASE-CDT-PUB-CALITP-P-001" / Resource Group "RG-CDT-PUB-SHRD-W-P-001"): web.AppServiceEnvironmentsClient#Get: Failure responding to request: StatusCode=403 -- Original Error: autorest/azure: Service returned an error. Status=403 Code="AuthorizationFailed" Message="The client 'kegan@compiler.la' with object id '[REDACTED]' does not have authorization to perform action 'Microsoft.Web/hostingEnvironments/read' over scope '/subscriptions/[REDACTED]/resourceGroups/RG-CDT-PUB-SHRD-W-P-001/providers/Microsoft.Web/hostingEnvironments/ASE-CDT-PUB-CALITP-P-001' or the scope is invalid. If access was recently granted, please refresh your credentials."

### CODE REVIEW

**Comment by thekaveman** [comment]
The [`plan` succeeded](https://calenterprise.visualstudio.com/CDT.OET.CAL-ITP/_build/results?buildId=34622&view=logs&j=bc150913-bfac-577d-ab97-4eb22da69064)

**Comment by afeld** [comment]
Made a suggestion to their docs to make that more obvious: https://github.com/hashicorp/terraform-provider-azurerm/pull/17694

**Comment by thekaveman** [comment]
Updated `plan`:

image

File: terraform/app_service.tf [hunk 1]
```diff
@@ -4,6 +4,7 @@ resource "azurerm_service_plan" "main" {
   resource_group_name = data.azurerm_resource_group.prod.name
   os_type             = "Linux"
   sku_name            = "P2v2"
+  app_service_environment_id = azurerm_app_service_environment_v3.main.id
```
Comment (afeld) [inline terraform/app_service.tf:7]: Not a blocker, but looks like this file didn't get formatted.

**Review by afeld — APPROVED** [review]

**Comment by afeld** [comment]
After deploy, make sure to confirm that the cipher suite isn't showing up, following the instructions in https://github.com/cal-itp/benefits/issues/771.

**Review by afeld — APPROVED** [review]

**Comment by thekaveman** [comment]
❌Nope: https://calenterprise.visualstudio.com/CDT.OET.CAL-ITP/_build/results?buildId=34629&view=logs&j=bc150913-bfac-577d-ab97-4eb22da69064&t=5c01d50c-3bf9-589e-956a-833a2240feb3

**Pull request merged**